settings.yaml

```diff
@@ -549,7 +549,17 @@
   Description: Indicates the maximum duration in seconds for which the inertial system will dead reckon.
   Notes:       The default value of 600 seconds was chosen as the expected duration for which the Duro Inertial solution can maintain sub-meter accuracy.
 
-- group: ins
+  group: ins
+  name: dr_timeout_pos_stddev
+  expert: false
+  type: double
+  units: meters
+  default value: '20'
+  readonly: false
+  Description: Indicates the maximum standard deviation of position for which the inertial system will dead reckon.
+  Notes:       The default value of 20 meters was chosen as the logical minimum standard of the position accuracy during dead reckon mode. 
```

Comment:
Can you remove the tab here and in the notes for the dr_duration_max. It does not seem to affect the console but seems unnecessary.